lib/semian.rb

```diff
@@ -166,6 +167,32 @@ def destroy(name)
   def resources
     @resources ||= {}
   end
+
+  private
+
+  def create_circuit_breaker(name, **options)
+    circuit_breaker = options.fetch(:circuit_breaker, true)
+    return unless circuit_breaker
+
+    exceptions = options[:exceptions] || []
+    CircuitBreaker.new(
+      name,
+      success_threshold: options[:success_threshold],
+      error_threshold: options[:error_threshold],
+      error_timeout: options[:error_timeout],
```

Comment:
Previously we'd raise from `#register` if these options weren't given (they have an empty value). Can we raise an `ArgumentError` if `circuit_breaker` is passed, but the options are not given?